README.rst

```diff
@@ -105,6 +105,7 @@ Contents
    aiosmtpd/docs/controller
    aiosmtpd/docs/handlers
    aiosmtpd/docs/cli
+   aiosmtpd/docs/smtpd
```

Comment:
I would suggest calling this `migrating` since in #93 I'm using `smtp` to document the `SMTP` class, and it would be better not to confuse things.